csrankings-s.csv

```diff
@@ -1397,6 +1397,7 @@ Spyros Samothrakis,University of Essex,https://www.essex.ac.uk/people/samot88004
 Spyros Voulgaris,VU Amsterdam,http://www.cs.vu.nl/~spyros,M_tdQJAAAAAJ
 Sravan Danda,BITS Pilani-Goa,http://www.bits-pilani.ac.in/goa/dandas/profile,5kSGWFoAAAAJ
 Srdjan Capkun,ETH Zurich,http://www.syssec.ethz.ch/people/capkun,VtkPBNsAAAAJ
+Srdjan Matic,IMDEA Software Institute,https://software.imdea.org/people/srdjan.matic,BQKUBJsAAAAJ

```

Comment:
Listed as "researcher", not "research professor". Please clarify in light of the inclusion criteria.

Reply:
Feel free to re-open if appropriate.